recipe/meta.yaml

```
@@ -50,10 +50,17 @@ test:
     - hypothesis >=3.58
     - pytest >=6.0
     - pytest-xdist
+    - package-testing
   imports:
     - pandas
+  source_files:
+    - package-testing/anaconda_package_tests
   commands:
     - pip check
+    - pytest
+    - pytest -s anaconda_package_tests
+
+
```

Requires specifying the path where pytest can find the packaged tests, see also https://github.com/anaconda-distribution/package-testing/blob/main/recipe/meta.yaml#L34
```suggestion
    - pytest -s -vv ${CONDA_PREFIX}/lib/python${PY_VER}/site-packages/anaconda_package_tests
```

---

Sure , since the folder is added . thought providing the directory name would suffice. Now will add the full path , Thanks.